src/actions/transformMatch.js

```diff
@@ -211,6 +211,16 @@ function transformMatch(m) {
     }
     newPlayer.buybacks = (player.buyback_log || []).length;
     newPlayer.total_gold = (player.gold_per_min * m.duration) / 60;
+    if (m.game_mode === 18 && Object.prototype.hasOwnProperty.call(player, 'ability_upgrades_arr')) {
+      const arr = [];
+      arr[0] = 1;
```

Comment:
instead of creating a dummy element here, perhaps just refer to the `i+1`th element in the column definition?

Reply:
I did it, just referring the the -1 index of incoming array.. let me know if that is a bad soln.